Migrate community chat to Discord (#2926)
tenzir/vast: web/blog/from-slack-to-discord/index.md

---
title: From Slack to Discord
description: Moving our community chat from Slack to Discord
authors: mavam

> image: slack-to-discord.light.png

Author: ```suggestion
image: /blog/from-slack-to-discord/slack-to-discord.light.png
```

@ditya I also tried this to get the image render in the main page carousel. Not sure if it makes more sense to just remove this metadata item, or whether there was a quick fix to make it work.

Reviewer: Pushed the quick fix.

Author: That works, but is the idea now that we pollute `/img` for blog post images?

Reviewer: I say for now, yes. An overhaul of the docs site is planned, and I say we don't spend more time on patching it before then.

Reviewer 2: Did I miss something? What's the planned overhaul for the docs site?

Author: Taking this discussion offline.

date: 2023-02-09
tags: [community, chat, discord]
---

We are moving our community chat from Slack to Discord. Why? TL;DR: because
Discord has better support for community building. VAST is not the first project
that abandons Slack. [Numerous][meilisearch] [open-source][appwrite]
[projects][deepset] [have][sst] [done][qovery] [the][neo4j] [same][discord-oss].

[meilisearch]: https://blog.meilisearch.com/from-slack-to-discord-our-migration/
[appwrite]: https://appwrite.io/
[deepset]: https://www.deepset.ai/blog/migration-to-discord
[sst]: https://sst.dev/blog/moving-to-discord.html
[qovery]: https://www.qovery.com/blog/feedback-from-slack-to-discord-13-months-later
[neo4j]: https://neo4j.com/blog/neo4j-community-is-migrating-from-slack-to-discord/
[discord-oss]: https://discord.com/open-source

<!--truncate-->

![Slack-to-Discord](/img/slack-to-discord.light.png#gh-light-mode-only)
![Slack-to-Discord](/img/slack-to-discord.dark.png#gh-dark-mode-only)

:::info Discord Invite Link
You can join our Discord community chat via <https://vast.io/discord>.
:::

> Here are our top reasons why we switched:

Reviewer: ```suggestion
Here are the top four reasons why we are switching:
```

- **Retention**: Slack's free plan has only 90 days message retention. We prefer
  permanence of our community discussion.

- **Moderation**: Discord has solid moderation tools that rely on role-based
  access, and makes it possible adhere to our [Code of
  Conduct](/docs/contribute/code-of-conduct) upon joining.

- **Invitation**: Unlimited invite links that do not expire.

> - **Inclusion**: users can self-assign their preferred pronouns.

Reviewer: ```suggestion
- **Inclusion**: Users can self-assign their preferred pronouns.
```

We hope that the majority of our Slack users understand these concerns and

> will rejoice with us over at Discord. See you there!

Reviewer: ```suggestion
will join us at Discord. See you there!
```
